lua/entities/gmod_wire_cpu.lua

```diff
@@ -4,8 +4,17 @@ ENT.PrintName       = "Wire ZCPU"
 ENT.Author          = "Black Phoenix"
 ENT.WireDebugName	= "ZCPU"
 
+
+
 if CLIENT then return end -- No more client
 
+local cpu_max_frequency = 1400000
+CreateConVar("wire_cpu_max_frequency", cpu_max_frequency, FCVAR_REPLICATED)
```

Comment:
Now make this ConVar a local variable, and call it `wire_cpu_max_frequency`.
Then use it on the line 15 (instead of calling `GetConVar("wire_cpu_max_frequency")`).